superset/tasks/cache.py

```diff
@@ -96,6 +96,7 @@ class DummyStrategy(Strategy):  # pylint: disable=too-few-public-methods
     def get_payloads(self) -> list[dict[str, int]]:
         session = db.create_scoped_session()
         charts = session.query(Slice).all()
+        session.close()
```

Comment:
I gather these standalone scoped sessions exist because tasks fall outside the jurisdiction of Flask. If this isn't the case then—to adhere to the KISS priniciple—we should simply reuse the Flask-SQLAlchemy session.